src/hooks/useInfoTokens.ts

```diff
@@ -39,16 +39,39 @@ export function useInfoTokens(
     }
   );
 
-  const indexPricesUrl = getServerUrl(chainId, "/prices");
-  // const indexPricesUrl = `https://pricing.mycelium.xyz/prices?network=${chainId}`;
+  const oldIndexPricesUrl = getServerUrl(chainId, "/prices");
+  const newIndexPricesUrl = `https://pricing.mycelium.xyz/prices?network=${chainId}`;
 
-  const { data: indexPrices } = useSWR([indexPricesUrl], {
+  const { data: newIndexPrices } = useSWR([newIndexPricesUrl], {
     // @ts-ignore spread args incorrect type
     fetcher: (...args) => fetch(...args).then((res) => res.json()),
     refreshInterval: 500,
     refreshWhenHidden: true,
   });
 
+  const { data: oldIndexPrices } = useSWR([oldIndexPricesUrl], {
+    // @ts-ignore spread args incorrect type
+    fetcher: (...args) => fetch(...args).then((res) => res.json()),
+    refreshInterval: 500,
+    refreshWhenHidden: true,
+  });
+
+  let indexPrices: Record<string, BigNumber> = {};
+  if (newIndexPrices && oldIndexPrices) {
```

Comment:
this condition will cause no prices to be used if one endpoint fails. we should make it so that if only one succeeds it uses thats, and only perform stale checks/overwrite if both are available